alphas/alpha_functions.py

```diff
@@ -29,3 +30,51 @@ def alpha_001(data: pd.DataFrame, ticker: str) -> pd.DataFrame:
     drop_signal_indices = data["actively_traded"].where(data["actively_traded"] == False).dropna().index
     raw_signal.loc[drop_signal_indices] = 0
     return raw_signal
+
+def alpha_006(data: pd.DataFrame) -> pd.Series:
+    """
+    div{inv[delta(bscore(90),14)],tsrank(delta{bscore(13),36},38)}
+    """
+    neg_bscore_delta = utils.neg(utils.delta(utils.bscore(data['Adj Close'], 90), 14))
+    tsrank_bscore_delta = utils.ts_rank(utils.delta(utils.bscore(data['Adj Close'], 13), 36), 38)
+
+    result = neg_bscore_delta/tsrank_bscore_delta
```

Comment:
You don't need to declare a variable immediately before you return the value. For eg, ` return neg_bscore_delta/tsrank_bscore_delta` works perfectly fine